### PR TITLE
Fix some other reference issues

### PR DESCRIPTION
**Pull request by agahkarakuzu**

### CODE REVIEW

File: content/index.ipynb [hunk 1]
```diff
@@ -659,20 +659,20 @@
...
     "\n",
-    "Compared to CSA averaged from C2-C3 vertebral levels measured from the UK Biobank database (N = 804) of 66.4 ± 6.61 mm<sup>2</sup> {cite:p}`Bedard2022-tr`, we measured a larger CSA of 73.59 ± 7.41 mm<sup>2</sup> for C2 vertebral level and 74.32 ± 7.91 mm<sup>2</sup> for C3 vertebral level. This might be caused by the fact that we used T2-w images, whilst Bédard and Cohen-Adad used T1-w images {cite:p}`Bedard2022-tr`. It has been shown that T2-w scans generally provide larger CSA compared to T1-w scans {cite:p}`Cohen-Adad2021-tn`.\n",
+    "Compared to CSA averaged from C2-C3 vertebral levels measured from the UK Biobank database (N = 804) of 66.4 ± 6.61 mm<sup>2</sup> {cite:p}`Bedard2022-tr`, we measured a larger CSA of 73.59 ± 7.41 mm<sup>2</sup> for C2 vertebral level and 74.32 ± 7.91 mm<sup>2</sup> for C3 vertebral level. This might be caused by the fact that we used T2-w images, whilst Bédard and Cohen-Adad used T1-w images {cite:p}`Bedard2022-tr`. It has been shown that T2-w scans generally provide larger CSA compared to T1-w scans {cite:p}`Bedard2022-tr`.\n",
```
Comment (valosekj) [inline content/index.ipynb]: Should be Cohen-Adad, 2021, not Bedard 2022

File: paper.bib [hunk 2]
```diff
@@ -178,47 +178,16 @@ @ARTICLE{Smith2021-mh
...
-              Yiannakas, Marios C and Weber, II, Kenneth A and Weiskopf,
-              Nikolaus and Wise, Richard G and Wyss, Patrik O and Xu, Junqian",
-  journal  = "Scientific Data",
-  volume   =  8,
-  number   =  1,
-  doi = {10.1038/s41597-021-01044-0},
-  pages    = "219",
-  month    =  dec,
-  year     =  2021,
+@article{Cohen2021-tn,
```
Comment (valosekj) [inline paper.bib]: Arent you mixing `10.1038/s41597-021-01044-0` and `10.1038/s41596-021-00588-0` into a single ref by this?

File: content/index.ipynb [hunk 3]
```diff
@@ -103,9 +103,9 @@
     "\n",
     "## 2.1 Participants\n",
     "\n",
-    "Two hundred and three participants from the _spine-generic_ dataset (multi-subject) were used {cite:p}`Cohen-Adad2021-tn`. The dataset is open-access, organized according to the Brain Imaging Data Structure (BIDS) standard {cite:p}`Gorgolewski2016-xf,Karakuzu2022-bids` and managed using [git-annex](href=\"https://git-annex.branchable.com) in [this GitHub repository](https://github.com/spine-generic/data-multi-subject/tree/r20230223).\n",
+    "Two hundred and three participants from the _spine-generic_ dataset (multi-subject) were used {cite:p}`Cohen2021-tn`. The dataset is open-access, organized according to the Brain Imaging Data Structure (BIDS) standard {cite:p}`Gorgolewski2016-xf,Karakuzu2022-bids` and managed using [git-annex](href=\"https://git-annex.branchable.com) in [this GitHub repository](https://github.com/spine-generic/data-multi-subject/tree/r20230223).\n",
```
Comment (valosekj) [inline content/index.ipynb]: Maybe `CohenAdad2021` would be better `Cohen2021`?

File: paper.bib (hunk 2 above)
Comment (agahkarakuzu) [reply paper.bib]: Are not they different versions of the same article?

File: content/index.ipynb (hunk 3 above)
Comment (agahkarakuzu) [reply content/index.ipynb]: These keys don't matter much, they are not rendered in the end.

File: paper.bib (hunk 2 above)
Comment (valosekj) [reply paper.bib]: No!

`10.1038/s41596-021-00588-0` is:
Cohen-Adad, J., Alonso-Ortiz, E., Abramovic, M. et al. Generic acquisition protocol for quantitative MRI of the spinal cord. Nat Protoc 16, 4611–4632 (2021). https://doi.org/10.1038/s41596-021-00588-0

while `10.1038/s41597-021-01044-0` is:
Cohen-Adad, J., Alonso-Ortiz, E., Abramovic, M. et al. Open-access quantitative MRI data of the spinal cord and reproducibility across participants, sites and manufacturers. Sci Data 8, 219 (2021). https://doi.org/10.1038/s41597-021-00941-8

And actually, `10.1038/s41597-021-01044-0` is a correction of `10.1038/s41597-021-00941-8`

**Comment by agahkarakuzu** [comment]
OK @valosekj these two commits should have fixed it.

**Comment by valosekj** [comment]
LGTM, thanks! I fixed the DOI also in REDME.md and content/index.ipynb in https://github.com/valosekj/PAM50-normalized-metrics-paper/pull/3/commits/2c732d21ae1e708734053783ff78c1d2aa04e9da.

**Pull request merged**